Documentation/Security/GuidelinesAdministrators/DirectoryIndexing.rst

```diff
@@ -1,111 +1,95 @@
+======================================
 
-Apache web server
-=================
+This applies to Apache web servers, especially in shared hosting environments
+where configuration is done via :file:`.htaccess` files.
 
-By removing the `Indexes` from `Options` (or not setting it in the first place),
-Apache does not show the list of files and directories.
+In Apache, directory indexing is controlled by the `Indexes` flag within the
+`Options` directive.
 
-In TYPO3, the default :file:`.htaccess` already contains the
-directive to disable directory indexing. Check if the following is
-in your :file:`.htaccess`:
+TYPO3's default :file:`.htaccess` disables indexing with the following setting:
 
-.. code-block:: apacheconf
-   :caption: /var/www/myhost/public/.htaccess
+..  code-block:: apacheconf
+    :caption: /var/www/myhost/public/.htaccess
 
-   # Make sure that directory listings are disabled.
-   <IfModule mod_autoindex.c>
+    <IfModule mod_autoindex.c>
       Options -Indexes
-   </IfModule>
+    </IfModule>
 
-This example, does not set all `Options`, it just removes `Indexes` from the
-list of Options. Directory indexing is provided by the module `autoindex`.
-By setting the options this way, it will be disabled in any case, even if the
-module is currently not active but might be activated at a later time.
+Alternatively, set this directly in your Apache site configuration:
 
-It is also possible, to configure the `Options` in the Apache configuration,
-for example:
+..  code-block:: apacheconf
+    :caption: /etc/apache2/sites-available/myhost.conf
 
-.. code-block:: apacheconf
-   :caption: /etc/apache2/sites-available/myhost.conf
-
-   <IfModule mod_autoindex.c>
+    <IfModule mod_autoindex.c>
       <Directory /var/www/myhost/public>
-         # override all Options, do not activate Indexes for security reasons
          Options FollowSymLinks
       </Directory>
-   </IfModule>
+    </IfModule>
 
-Please note that the `Options` directive can be
-used in several containers (for example `<VirtualHost>`, `<Directory>`,
-in the Apache configuration) or in the file :file:`.htaccess`.
-Refer to the `Options <https://httpd.apache.org/docs/2.4/mod/core.html#options>`__
-directive for more information.
+See the `Apache Options directive documentation <https://httpd.apache.org/docs/2.4/mod/core.html#options>`__
+for more information.
 
-Nginx
-=====
+..  _security-directory-indexing-nginx:
 
-For Nginx, directory listing is handled by the `ngx_http_index_module` and
-directory listing is disabled by default.
+Disable indexing in Nginx (server block)
+========================================
 
-You can explicitly disable directory listing by using the parameter
-`autoindex`.
+This applies to Nginx installations where settings are configured in the
+server block (virtual host configuration).
 
-.. code-block:: nginx
-   :caption: /etc/nginx/sites-available/myhost.com
+Although directory listing is disabled by default in Nginx, you can explicitly
```

Comment:
Directory Listing is deactivated in default `htaccess` of TYPO3, too:
https://github.com/TYPO3/typo3/blob/main/typo3/sysext/install/Resources/Private/FolderStructureTemplateFiles/root-htaccess#L368-L371

Reply:
That is what we are writing in the Apache section. However, Apache itself does not disable directorty listing by default while nginx  does that.